pkg/lease/v1alpha1/csp_lease_test.go

```diff
@@ -18,16 +18,17 @@ package lease
 import (
 	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
 
+	"os"
+	"strconv"
+	"testing"
+
 	"github.com/golang/glog"
 	openebs "github.com/openebs/maya/pkg/client/generated/clientset/internalclientset"
 	openebsFakeClientset "github.com/openebs/maya/pkg/client/generated/clientset/internalclientset/fake"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
IMO this is not needed since alias & last folder are same

Reply:
Better to use `corev1` as alias

Reply:
done